src/components/material/periodical/helper.ts

```diff
@@ -28,4 +40,71 @@ export function makePeriodicalEditionsFromHoldings(holdings: HoldingsV3[]) {
     .flat();
 }
 
+export function filterAndSortPeriodicalEditions(baseData: {
+  [key: string]: PartialPeriodicalEdition[];
+}) {
+  const periodicalEditions = Object.entries(baseData);
+  const filteredEditions = periodicalEditions.map((yearAndEditions) => {
+    return yearAndEditions[1].reduce((acc, edition) => {
+      if (!edition.volumeNumber) {
+        return acc;
+      }
+      const includesValueAlready = acc.includes(edition.volumeNumber);
+      if (!includesValueAlready) {
+        acc.push(edition.volumeNumber);
+      }
+      return acc;
+    }, [] as string[]);
+  });
+  const allYears = periodicalEditions.map(
+    (yearEditionPair) => yearEditionPair[0]
+  );
+  const filteredPeriodicalEditionsObj = allYears.reduce((acc, curr, index) => {
+    // Sort editions array
+    // eslint-disable-next-line no-param-reassign
+    acc[curr] = filteredEditions[index].sort((a, b) => {
+      return a.localeCompare(b, "da-DK", { numeric: true });
+    });
+    return acc;
+  }, {} as { [key: string]: string[] });
+  return filteredPeriodicalEditionsObj;
+}
```

Comment:
I do not understand what is going on here. We should talk this through face to face.

Reply:
Summary of the talk: 
There is a general need for utility functions that work with objects, as the current out of the box object value filtering possibilities are not ideal, the produced code can be quite confusing and the need for them spans around the whole project. As such, we decided to add the full lodash package to the project. We previously agreed on only importing specific parts of the library, but now found out that lodash themselves don't recommend that (link for reference: https://lodash.com/per-method-packages) and this method of working with lodash would increase the bundle size of the project. Instead, we limit the bundle size by using the lodash-recommended package - babel-plugin-lodash that only imports specific methods used in components. Please refer to the link above for further explanation.

Reply:
fixed 😉